Cards/Resource/Card_Energy.tres

```diff
@@ -0,0 +1,27 @@
+[gd_resource type="Resource" script_class="CardBase" load_steps=8 format=3 uid="uid://ckaua56xmrxib"]
+
+[ext_resource type="Script" path="res://Cards/Effects/EffectAddEnergy.gd" id="1_3yh17"]
+[ext_resource type="Script" path="res://Cards/Effects/EffectData.gd" id="2_dr7f0"]
+[ext_resource type="Script" path="res://Cards/EnergyData.gd" id="3_2l1ax"]
+[ext_resource type="Script" path="res://Cards/CardBase.gd" id="3_opvsg"]
+
+[sub_resource type="Resource" id="Resource_jur0i"]
+script = ExtResource("1_3yh17")
+
+[sub_resource type="Resource" id="Resource_6gvxm"]
+script = ExtResource("2_dr7f0")
+effect = SubResource("Resource_jur0i")
+value = 70
+target_type = 0
```

Comment:
this is not insta-cast
things will have to change with the targeting PR anyway, i will rewrite this card once everything is merged I think

Reply:
Just for reference, what is insta-cast?

Reply:
Insta-cast is the mechanic that makes it so you don't need to target someone when it's clear who the card is for (buff, heal, damage everyone etc...)
This changed due to the targeting PR that was just merged, ask me if you don't understand the new system